index.js

```diff
@@ -32,6 +32,12 @@ hexo.extend.processor.register(hexo.config.image_sizes.pattern, function (file)
 // record when a post uses an image and embed the image in the post.
 hexo.extend.tag.register("imsize", imsizeTag, {ends: true});
 
+// Switch the slashes in case we get Windows-style paths.
+function switchSlashes(str)
+{
+  return str.split('\\').join('/');
```

Comment:
This might break paths on macOS and Linux, which allow paths to have backslashes as escapes for spaces: `/home/user/My\ File` is a valid path there, and needs the backslash.

Reply:
I think we can avoid that issue by normalizing the path first:

```suggestion
  return path.normalize(str).replace(new RegExp(path.sep, 'g'), '/');
```